2022/covid-19.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: JBCNConf - COVID-19 Policy
+bodyClass: home page-template-default page page-id-7 eventstation-class  eventstation-shop-column-4 wpb-js-composer js-comp-ver-5.0.1 vc_responsive
+---
+{% include base.html %}
+<div class="content-heading">
+  <div class="container-fluid">
+    <div class="heading-navigation">
+      <i class="fa fa-address-book"></i>
+      <ul>
+        <li>Home</li>
+        <li>&#47;</li>
+        <li>COVID-19 Policy</li>
+      </ul>
+    </div>
+
+    <div class="heading-title">
+    </div>
+  </div>
+</div>
+<div class="site-sub-content clearfix">
+  <div class="container-fluid">
+    <div class="row alternative-row">
+      <div class="col-lg-9 col-sm-9 col-xs-12 site-content-left">
+        <div class="category-post-list post-list single-list">
+          <article
+            class="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment"
+            id="post-294">
+            <div class="post-wrapper">
+              <div class="post-header">
+                <ul class="post-information">
+
+                </ul>
+                <h2 style="margin-top:50px;">COVID-19 Policy</h2>
+                For all our events, including conferences, workshops, and social events, we take the COVID-situation
+                seriously. We want to reduce the risk of infection as much as possible, and keep everyone safe.
+
+                We will adhere to <a
```

Comment:
I would include an explicit message saying something like: 
`Here you can check the local measures that apply to us on the basis of our government in Catalonia`